enhancements/two-nodes-openshift/2no.md

```diff
@@ -178,29 +210,37 @@ This will be achieved by having the same entity that drives the configuration of
 
+The plan is for this to be changed by one of the nodes during pacemaker initialization. Pacemaker initialization should be initiated by CEO when it detects that the cluster controlPlane topology is set to `DualReplica`.
+
+#### 2NO Resource Agent Remediation Operator
+
+From a high level, the proposed in-cluster operator's job is to ensure that the RHEL-HA resource agents that need to be running to have a safe 2-node cluster are healthy.
+The most involved aspect of this is collecting BMC credentials, and ensuring that the RHEL-HA components can sync them to disk during initialization or whenever these secrets are updated.
+
 ### Topology Considerations
 
 2NO represents a new topology and is not appropriate for use with HyperShift, SNO, or MicroShift
 
 #### Standalone Clusters
 
-Two-node OpenShift is first and foremost a topology of OpenShift, so it should be able to run without any assumptions of a cluster manager.
+Two-node OpenShift is first and foremost a topology of OpenShift, so it should be able to run without any assumptions of a cluster manager. To achieve this, we will need to enable the installation
+of two-node clusters via the Agent-Based Installer to ensure that we are still meeting the installation requirement of using only 2 nodes.
 
 ### Implementation Details/Notes/Constraints
 
-While the target installation requires exactly 2 nodes, this will be achieved by proving out the "bootstrap plus 2 nodes" flow in the core installer and then using the Assisted Installer's ability to bootstrap from one of the target machines to remove the requirement for a bootstrap node.
+While the target installation requires exactly 2 nodes, this will be achieved by proving out the "bootstrap plus 2 nodes" flow in the core installer and then using assisted-service-based installers to bootstrap from one of the target machines to remove the requirement for a bootstrap node.
 
 So far, we've discovered topology-sensitive logic in ingress, authentication, CEO, and the cluster-control-plane-machineset-operator. We expect to find others once we introduce the new infrastructure topology.
 
 The delivery of RHEL-HA components will be opaque to the user and be delivered as an [MCO Extension](../rhcos/extensions.md) in the 4.18 and 4.19 timeframes.
 A switch to [MCO Layering](../ocp-coreos-layering/ocp-coreos-layering.md ) will be investigated once it is GA in a shipping version of OpenShift.
 
-Once installed, the configuration of the RHEL-HA components will be done via an in-cluster entity. This entity could be a dedicated in-cluster operator or a function of CEO triggering a script on one of the control-plane nodes. This initialization will require that RedFish details have been collected by the installer.
+Once installed, the configuration of the RHEL-HA components will be done via an in-cluster entity. This entity could be a dedicated in-cluster operator or a function of CEO triggering a script on one of the control-plane nodes.
```

Comment:
I think CEO will not be able to run any script on a host:
https://github.com/openshift/cluster-etcd-operator/blob/master/manifests/0000_20_etcd-operator_06_deployment.yaml#L28-L29

also not sure that the security team is happy to give another operator root privs

Reply:
Added a note to cover the risk of privilege escalation.